src/comm/LinkManager.cc

```diff
@@ -185,6 +186,19 @@ SharedLinkInterfacePtr LinkManager::mavlinkForwardingLink()
     return nullptr;
 }
 
+SharedLinkInterfacePtr LinkManager::mavlinkForwardingSupportLink()
+{
+    for (int i = 0; i < _rgLinks.count(); i++) {
```

Comment:
Please use range based loop over manual index.
https://en.cppreference.com/w/cpp/language/range-for

Reply:
Thanks for the suggestion, I am used to this manual index approach. Just out of curiosity, is there any benefit on the range based for versus manual index? Or is it just that c++ is evolving into that?

I reused that scheme used all over linkManager class. Maybe we should fix it in all the places in a separate PR?